Jellyfin.Server/Program.cs

```diff
@@ -82,7 +82,7 @@ private static async Task StartApp(StartupOptions options)
 
             appConfig = await CreateConfiguration(appPaths).ConfigureAwait(false);
 
-            await CreateLogger(appConfig, appPaths).ConfigureAwait(false);
+            CreateLogger(appConfig, appPaths);
```

Comment:
I think this was specifically changed before, why revert?

Reply:
The async code in the CreateLogger method was all in config creation, this was all moved to the new awaited CreateConfiguration method. This left CreateLogger synchronous so it is not awaitable.